MobileCenterDistribute/MobileCenterDistributeTests/MSDistributeTests.m

```diff
@@ -65,8 +65,6 @@ - (void)setUp {
   [MSKeychainUtil clear];
   self.sut = [MSDistribute new];
 
-  // TODO: Add unit tests for MSBasicMachOParser.
-  // FIXME: MSBasicMachOParser don't work on test projects. It's mocked for now to not fail other tests.
```

Comment:
The comment is still almost valid, it's just permanent now so you can still add this comment: `// MSBasicMachOParser may fail on test projects' main bundle. It's mocked to prevent it.`